Solutions/CrowdStrike Falcon Endpoint Protection/Data Connectors/CrowdStrikeFalconAdversaryIntelligence/azure_deploy_CrowdstrikeFalconIndicatorsofCompromise.json

```diff
@@ -213,7 +213,7 @@
                         "AAD_CLIENT_SECRET": "[parameters('AadClientSecret')]",
                         "LOOK_BACK_DAYS": "[parameters('LookBackDays')]",
                         "WORKSPACE_ID": "[parameters('WorkspaceId')]",
-                        "WEBSITE_RUN_FROM_PACKAGE": "https://aka.ms/sentinel-CrowdStrikeFalconIOC-Functionapp"
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://aka.ms/sentinel-CrowdStrikeFalconAdversaryIntelligence-Functionapp"
```

Comment:
@v-prasadboke Question on this, I've derived this value based on what the format I see in other data connectors. WiIl this work?

Reply:
It was good earlier, No worries for the shortlink. I'll update the hyperlink from our shortlink portal